portalnet/src/overlay_service.rs

```diff
@@ -1322,6 +1370,8 @@ where
             {
                 debug!(%err, cid.send, cid.recv, peer = ?cid.peer.client(), content_keys = ?content_keys_string, "unable to process uTP payload");
             }
+            // explictically drop semaphore permit in thread so it gets moved into it
```

Comment:
`explicitly` ... but also i'm not sure what you mean by `so it gets moved into it`, so what gets moved into what?

Reply:
so that permit gets moved into the new thread.

Reply:
the question is not so much for me, but more to update the comment with the answer for whoever is reading the code in the future and might have the same question

Reply:
oh I get what you mean now `it` isn't a descriptive word of the object in this case the permit fair enough :+1: